packages/components/psammead-episode-list/src/MediaIndicator.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string } from 'prop-types';
+
+const Wrapper = styled.div`
+  display: inline-block;
+  width: ${props => props.size};
+  height: ${props => props.size};
+`;
+
+const MediaIndicator = ({ size }) => (
+  <Wrapper
+    aria-hidden="true"
```

Comment:
An attribute of focusable false is also needed as IE11 makes all SVG's focusable by default